config/migrations/2023/20230726100700-fix-agb-names.sparql

```diff
@@ -0,0 +1,65 @@
+DELETE {
+  GRAPH <http://mu.semte.ch/graphs/administrative-unit> {
+    <http://data.lblod.info/id/bestuurseenheden/e852a0c8-e610-4359-9d8f-baa42e1bf5cf> <http://www.w3.org/2004/02/skos/core#prefLabel> "AGB Begijnendijk ".
```

Comment:
You used the same uuid in the entire migration :grimacing: 

![image](https://github.com/lblod/app-organization-portal/assets/23281659/7f043d55-956c-4613-9f93-5e266a7ea1bf)